libcst/codemod/commands/remove_unused_imports.py

```diff
@@ -4,9 +4,18 @@
 # LICENSE file in the root directory of this source tree.
 #
 
-from libcst import Import, ImportFrom
-from libcst.codemod import VisitorBasedCodemodCommand
-from libcst.codemod.visitors import RemoveImportsVisitor
+from typing import Set, Tuple, Union
+
+from libcst import Import, ImportFrom, ImportStar, Module
+from libcst.codemod import CodemodContext, VisitorBasedCodemodCommand
+from libcst.codemod.visitors import GatherCommentsVisitor, RemoveImportsVisitor
+from libcst.helpers import get_absolute_module_for_import
+from libcst.metadata import PositionProvider, ProviderT
+
+
+DEFAULT_SUPPRESS_COMMENT_REGEX = (
+    r".*\W(noqa|lint-ignore: ?unused-import|lint-ignore: ?F401)(\W.*)?$"
+)
```

Comment:
Also, don't forget about this one as well, I think it's better than the current one:

```py
DEFAULT_SUPPRESS_COMMENT_REGEX = (
    r".*\W(noqa[^:]|noqa: ?F401|lint-ignore: ?unused-import|lint-ignore: ?F401)(\W.*)?$"
)
```

Reply:
This doesn't match `# noqa` though, which is the most common case

Reply:
You are right!

What do you think of:

```py
r” .*\W(noqa|lint-ignore)(: ?(F401|unused-import)|(?= )|$)(\W.*)?$”
```